hera_pspec/pspecdata.py

```diff
@@ -1069,8 +1106,9 @@ def pspec(self, bls1, bls2, dsets, input_data_weight='identity', norm='I',
         dset1 = self.dsets[self.dset_idx(dsets[0])]
         dset2 = self.dsets[self.dset_idx(dsets[1])]
 
-        # get polarization array from zero'th dset
-        pol_arr = map(lambda p: pyuvdata.utils.polnum2str(p), dset1.polarization_array)
+        # check pol_select inputs
+        if pol_select == None:   
```

Comment:
This raises an error by default, as `pol_select=None` by default. If `pol_select` is not optional, it's best to not set a default value.

Reply:
Also, is it intentional to make the user always specify the polarization? I guess there isn't a sensible default behaviour?

Or, maybe, if there is only one polarisation in the datasets, should it just assume that it should use that polarisation?